homeassistant/components/amcrest/camera.py

```diff
@@ -51,6 +52,43 @@
 _SRV_TOUR_ON = "start_tour"
 _SRV_TOUR_OFF = "stop_tour"
 
+_SRV_PTZ_CTRL = "ptz_control"
+_ATTR_PTZ_TT = "travel_time"
+_ATTR_PTZ_MOV = "movement"
+_MOV = [
+    "zoom_in",
+    "zoom_out",
+    "right",
+    "left",
+    "up",
+    "down",
+    "right_down",
+    "right_up",
+    "left_down",
+    "left_up",
+    "bottom_right",
+    "top_right",
+    "bottom_left",
+    "top_left",
```

Comment:
It's really not necessary at all to have this duplicated.   I originally wrote this to use the same terminology in the API (right_up, left_down...) which seems more natural to me.   But then I later discovered that the Foscam component chose to match these directions to the names of their associated mdi icon arrows  (top_right, bottom_left...).

It was simple enough to chose to accept either term in the code.   If you think that is just not needed, then please pick which 4 terms you prefer, and we'll just go with that.

Reply:
I tend to think it makes more sense for the values of the `movement` parameter to be consistent. If we're using `left`, `right`, `up` and `down`, then the combined actions should use those terms rather than something different.

Reply:
That was my original thinking too and of course that matches the API.      I just now went the other way when examining the lovelace card for handling the movement.     I can easily see someone who uses Foscam (very popular before Amcrest split) getting an Amcrest and appreciate the consistency.    Really no big deal.   I can change it back.

Reply:
I kind of hate to ask, but please change it back. Besides the consistency with the other movement options, down & up are directions, whereas bottom and top are locations. And since this a parameter which indicates movement direction...

Reply:
ok - changed it back to how I had it in the first place!